### PR TITLE
[3.9][plugin][privacyconsent] - notify more than one

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue https://github.com/joomla-projects/privacy-framework/issues/243

### Summary of Changes
notify about expired consents more than one

### Testing Instructions
see https://github.com/joomla-projects/privacy-framework/issues/243

### CODE REVIEW

File: plugins/system/privacyconsent/privacyconsent.php [hunk 1]
```diff
@@ -588,7 +588,6 @@ private function remindExpiringConsents()
 					return false;
 				}
```
Comment (Quy) [inline plugins/system/privacyconsent/privacyconsent.php]: Delete blank line.

**Comment by farrelljsec** [comment]
I have tested this item :white_check_mark: successfully on 519756739e5b200099daa5a6e5f83932abe2e1c1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22028">issues.joomla.org/tracker/joomla-cms/22028</a>.</sub>

**Comment by sanderpotjer** [comment]
I have tested this item :white_check_mark: successfully on 519756739e5b200099daa5a6e5f83932abe2e1c1<br /><br />Came accross another issue here (https://github.com/joomla/joomla-cms/issues/22087) while testing, but not related. If that bug is fixed this is working as expected and sending all notifications instead of just 1.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22028">issues.joomla.org/tracker/joomla-cms/22028</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Pull request merged**